app/src/main/java/de/danoeh/antennapod/ui/screen/feed/FeedItemlistFragment.java

```diff
@@ -713,6 +721,14 @@ public void onKeyUp(KeyEvent event) {
         }
     }
 
+    private Pair<Integer, Integer> getScrollPosition() {
```

Comment:
Is it really necessary to create a new function that contains only one line and simply calls another function wirh the same name? I think this could be simplified by just calling the content directly